app/controllers/reservations_controller.rb

```diff
@@ -10,9 +10,29 @@ def show
   end
 
   def new
+    @date = Time.now.strftime("%B %d, %Y")
+    @pedalo = Pedalo.find(params[:pedalo_id])
+    @reservation = Reservation.new
+    # do some JS if time to display the total price
```

Comment:
this will have to be done in new.html.erb

Reply:
ok